code/_compile_options.dm

```diff
@@ -61,7 +61,7 @@ var/list/del_counter = list()
 
 #define MIN_COMPILER_VERSION 508
 #if DM_VERSION < MIN_COMPILER_VERSION //Update this whenever you need to take advantage of more recent byond features
-#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update.
+#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update. As for August of 2015, it uses the BETA version of BYOND.

```

Comment:
If you download 507 and change the min needed version to 507, the code
doesn't compile because I guess it takes advantages of features only
offered by the 508 beta.

On 6 August 2015 at 21:23, Alex <notifications@github.com> wrote:

> In code/_compile_options.dm
> <https://github.com/tgstation/-tg-station/pull/11052#discussion_r36395560>
> :
>
> > @@ -61,7 +61,7 @@ var/list/del_counter = list()
> >
> >  #define MIN_COMPILER_VERSION 508
> >  #if DM_VERSION < MIN_COMPILER_VERSION //Update this whenever you need to take advantage of more recent byond features
> > -#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update.
> > +#error Your version of BYOND is too out-of-date to compile this project. Go to byond.com/download and update. As for August of 2015, it uses the BETA version of BYOND.
>
> Can't you use the MIN_COMPILER_VERSION definition in the error message?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tgstation/-tg-station/pull/11052/files#r36395560>.
>